packages/wallets/walletconnect/src/adapter.ts

```diff
@@ -16,12 +17,11 @@ import {
 } from '@solana/wallet-adapter-base';
 import type { Transaction } from '@solana/web3.js';
 import { PublicKey } from '@solana/web3.js';
-import WalletConnectClient, { CLIENT_EVENTS } from '@walletconnect/client';
+import WalletConnectClient from '@walletconnect/sign-client';
```

Comment:
This adapter should be refactored to be a `Loadable` adapter -- the [Strike multisig](https://github.com/solana-labs/wallet-adapter/blob/master/packages/wallets/strike/src/adapter.ts#L14-L38) or [Torus adapters](https://github.com/solana-labs/wallet-adapter/blob/7612489924877c1cf5ff034c8eadb5d1311573ca/packages/wallets/torus/src/adapter.ts#L78-L90) are examples of this. Imports of types only, dynamic imports of actual SDKs. Ideally we can find a way to consume a single library that wraps any `@walletconnect/*` stuff needed so only one file load is required.